docs/whatsnew/0.8.rst

```diff
@@ -0,0 +1,140 @@
+
+???
+
+.. _whatsnew-0.8-coordinates:
+
+New coordinate package
+======================
+
+Spatial locations are now represented by SunPy coordinates.
+Coordinates can be defined very simply::
+
+    >>> from astropy import units as u
+    >>> from astropy.coordinates import SkyCoord
+    >>> from sunpy.coordinates import frames
+    >>> a = SkyCoord(200*u.arcsec, 300*u.arcsec, frame=frames.Helioprojective)
+    >>> a
+    <SkyCoord (Helioprojective: obstime=None, rsun=695508.0 km, observer=earth): (Tx, Ty) in arcsec
+    ( 200.,  300.)>
+    >>> b = SkyCoord(-20*u.degree, -56*u.degree, frame=frames.HeliographicStonyhurst)
+    >>> b
+    <SkyCoord (HeliographicStonyhurst: obstime=None): (lon, lat, radius) in (deg, deg, km)
+    (-20., -56.,  695508.)>
+
+
+The coordinate "a" is in the helioprojective coordinate system, and the coordinate "b"
+is in the heliographic Stonyhurst system.
+
+Maps can also be used to define coordinate frames::
+
+    >>> import sunpy.map
+    >>> import sunpy.data.sample
+    >>> aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    >>> c = SkyCoord(-45*u.arcsec, 600*u.arcsec, frame=aia_map.coordinate_frame)
+    >>> c
+    <SkyCoord (Helioprojective: obstime=2011-06-07 06:33:02.770000, rsun=696000000.0 m, observer=<HeliographicStonyhurst Coordinate (obstime=None): (lon, lat, radius) in (deg, deg, m)
+    ( 0.,  0.048591,   1.51846026e+11)>): (Tx, Ty) in arcsec
+    (-45.,  600.)>
+
+The coordinate "c" is now defined with respect to the coordinate frame derived from the map.
+The observer attribute::
+
+    >>> c.observer
+    <HeliographicStonyhurst Coordinate (obstime=None): (lon, lat, radius) in (deg, deg, m)
+    ( 0.,  0.048591,   1.51846026e+11)>
+
+defines the location from which the coordinate was observed.
+
+
+Transformation between solar physics coordinate systems
+-------------------------------------------------------
+
+Transformation between solar physics coordinate frames is simple::
+
+    >>> c.transform_to(frames.HeliographicStonyhurst)
+    <SkyCoord (HeliographicStonyhurst: obstime=2011-06-07 06:33:02.770000): (lon, lat, radius) in (deg, deg, km)
+    (-3.51257477,  39.27459767,  696000.00000088)>
+
+Transformation to astropy coordinate systems
+--------------------------------------------
+
+Solar physics coordinates can also be transformed into astrophysical coordinates.
+For example, to convert to the International Celestial Reference System (ICRS)::
+
+    >>> c.transform_to('icrs')
+    <SkyCoord (ICRS): (ra, dec, distance) in (deg, deg, km)
+    ( 240.91724178, -6.72107631,  369983.29218687)>
+
+Specification of observer at any major solar system body
+--------------------------------------------------------
+
+Major solar system bodies can be used to specify observer locations in SkyCoord::
+
+    >>> from sunpy.coordinates import get_body_heliographic_stonyhurst
+    >>> obstime = '2011-06-07 06:33:02'
+    >>> observer = get_body_heliographic_stonyhurst('Mars', obstime)
```

Comment:
you don't *need* use use `get_body_heliographic_stonyhurst` as you can just pass that string into the observer kwarg of `SkyCoord`.